src/utils/yaml-utils.ts

```diff
@@ -2,6 +2,26 @@ import yaml from "yaml"
 
 import { sanitizer } from "@services/utilServices/Sanitizer"
 
+const LINE_BREAK_REGEX = "[\r\n\x0B\x0C\u0085\u2028\u2029]"
```

Comment:
rather than trying to reimplement white space character check, i suggest only caring about new line characters and let `String.prototype.trim()` handle spaces. 

Currently, in the attempt of implementing both, the check for spaces is not as [complete](https://github.com/v8/v8/blob/7f55dc42841c93dbf095995ffd2f6a7ca41bf51e/src/builtins/string-trim.tq#L151) as the one implemented for the function call trim. 

Do we care about any space here? ie ` hi there ` should be `hithere`?

Reply:
Hmm the new line regex is really just to catch the new line characters, and not to replace other whitespace - I could replace it with `[\r\n\u2028\u2029]` instead. The in between spaces should be preserved